soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
If the ball obstacle causes constant replanning, isn't there something we can do on the planning side to fix it rather than here?

Consider the case where the ball is moving away from the robot.  Here's what your new code says to do:
* go directly towards a point on the otherside of the ball, ignoring the ball as an obstacle.
* if you get within the course approach radius of the ball, then count the ball as an obstacle and try go around it

It seems like we would get better paths if we just count it as an obstacle the whole time, right?  Am I misunderstanding what's going on?